dev/run-tests.py

```diff
@@ -128,7 +128,7 @@ def determine_tags_to_exclude(changed_modules):
     tags = []
     for m in modules.all_modules:
         if m not in changed_modules:
-            tags += m.test_tags
+            tags += m.tags
```

Comment:
@srowen Can you chime in on this? I wonder why we need to exclude the tags in this script. `excluded_tags = determine_tags_to_exclude(changed_modules)`